### ISSUE TITLE
Cannot write Hyphen on terminal

### ISSUE THREAD

**Issue by abdel-ely-ds** [jupyterlab/jupyterlab#11058]
I try to write someting like this git checkout -b branch on terminal but the hyphen is changed to 6 so I get git checkout 6b branch. 

My operating system is linux
Any help thank you

**Comment by Mithil467** [comment]
What shell are you using?
Try copy pasting the hyphen (-) in the terminal and see what is printed.
Can you confirm if hyphen works in some terminal other than in jupyterlab?

**Comment by abdel-ely-ds** [comment]
when i copy paste the hyphen it works correctly: it shows hypthen instead of the number 6. I confirm that hyphen works on other terminals

**Comment by abdel-ely-ds** [comment]
I tested with different browsers. It works on mozilla and not chrome

**Comment by Mithil467** [comment]
Can you confirm if it's not just a visual glitch but indeed is replacing the typed character? Which means that the command `git checkout -b branch_name` looks like `git checkout 6b branch_name` but does it treat it as a hyphen or a 6?

**Comment by abdel-ely-ds** [comment]
yes it's not. The command is not known

**Comment by Mithil467** [comment]
Can you try running `chrome --disable-extensions` and see if the error persists?

**Comment by abdel-ely-ds** [comment]
Yes i have always the same problem (sorry for this late response)

**Comment by eric-leblouch** [comment]
I get the same behavior on Chrome Version 94.0.4606.81 (Official Build) (64-bit) for all the characters that are numbers when shifted as well as other special characters. All alphabetical letters works correctly, not an issue of using the us layout instead of the french one.

No issue inside a notebook or text editor in the same JupyterLab instance.

**Comment by JasonWeill** [comment]
If this is still a problem, can you please share the type of keyboard you're using: language, country, and intended operating system?

**Comment by eric-leblouch** [comment]
It does work for me.
